### PR TITLE
Add styles to npm package

### PR DESCRIPTION
**Pull request by APJohns**

### CODE REVIEW

File: projects/epsilon-blueprint/src/lib/_global-variables.scss [hunk 1]
```diff
@@ -40,9 +40,11 @@ $font-size-small: 10px;
 $font-size-h1: 22px;
 $font-size-h2: 15px;
 $font-size-h3: 12px;
+$font-size-h4: 12px;
```
Comment (kaseyllabelle) [inline projects/epsilon-blueprint/src/lib/_global-variables.scss]: Can you rebase? These were from my last PR.

File: projects/epsilon-blueprint/src/styles.scss [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+}
+
+:focus,
+button:focus
+{
+    outline: none;
+    box-shadow: $box-shadow-focus-default;
+}
+
+/* grid overrides */
```
Comment (kaseyllabelle) [inline projects/epsilon-blueprint/src/styles.scss]: I wouldn't be mad if we created grid.scss and moved these there :)

File: src/styles/blueprint-ui-styles/typography.scss [hunk 3]
```diff
@@ -12,8 +12,7 @@
   }
 }
 
-
-.nav-secodnary
+.nav-secondary
```
Comment (kaseyllabelle) [inline src/styles/blueprint-ui-styles/typography.scss]: Can we make sure this typo doesn't exist in the component?

File: projects/epsilon-blueprint/src/lib/_global-variables.scss (hunk 1 above)
Comment (APJohns) [reply projects/epsilon-blueprint/src/lib/_global-variables.scss]: This is actually a duplicate file, removing it.

**Pull request merged**